app/config/security.yml

```diff
@@ -33,33 +33,36 @@ security:
             anonymous: true
 
         login_firewall:
-            pattern: ^/login$
+            pattern: "^%base_path%/login$"
             anonymous:  ~
 
         secured_area:
             pattern: ^/
             form_login:
                 provider: fos_userbundle
                 csrf_token_generator: security.csrf.token_manager
+                check_path: "%base_path%/login_check"
+                login_path: "%base_path%/login"
+                default_target_path: "%base_path%/"
 
             anonymous: true
             remember_me:
                 secret: "%secret%"
                 lifetime: 31536000
-                path: /
+                path: "%base_path%/"
                 domain: ~
 
             logout:
-                path:   /logout
-                target: /
+                path:   "%base_path%/logout"
+                target: "%base_path%/"
 
     access_control:
-        - { path: ^/api/doc, roles: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/api/version, roles: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/register, role: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/resetting, role: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: /(unread|starred|archive).xml$, roles: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/settings, roles: ROLE_SUPER_ADMIN }
-        - { path: ^/annotations, roles: ROLE_USER }
-        - { path: ^/, roles: ROLE_USER }
+        - { path: "^%base_path%/api/doc", roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "^%base_path%/api/version", roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "^%base_path%/login", roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "^%base_path%/register", role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "^%base_path%/resetting", role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "%base_path%/(unread|starred|archive).xml$", roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: "^%base_path%/settings", roles: ROLE_SUPER_ADMIN }
+        - { path: "^%base_path%/annotations", roles: ROLE_USER }
+        - { path: "^%base_path%/", roles: ROLE_USER }
```

Comment:
Thanks to the comment above about the extra route, I noticed that the path matched by ^$ isn't secured and causes an exception to be thrown. I will make sure this is fixed once we've decided what to do about the extra route.